assets/js/admin-pull.js

```diff
@@ -98,3 +88,115 @@ const getURL = () => {
 
 	return `${ baseURL }&pull_post_type=${ postType }&status=${ status }`;
 };
+
+document.addEventListener( 'DOMContentLoaded', async function () {
+	const container = document.querySelector( '.searchable-select' );
+	const inputContainer = container.querySelector(
+		'.searchable-select__input-container'
+	);
+	const input = container.querySelector( '.searchable-select__input' );
+	const icon = container.querySelector(
+		'.searchable-select__input-container > .dashicons-arrow-down'
+	);
+	const dropdown = container.querySelector( '.searchable-select__dropdown' );
+
+	const itemss = await fetch( '/wp-admin/admin-ajax.php', {
```

Comment:
Typeo? If it's meant to be searchResults or similar then use the longer name for clarity.

```suggestion
	const items = await fetch( '/wp-admin/admin-ajax.php', {
```

Reply:
Improved the name of var